crates/subnetworks/beacon/src/storage.rs

```diff
@@ -661,6 +677,28 @@ impl BeaconStorage {
     }
 }
 
+/// Prune old bootstrap data from the database
+fn prune_old_bootstrap_data(
+    sql_connection_pool: &Pool<SqliteConnectionManager>,
+) -> Result<(), ContentStoreError> {
+    let conn = sql_connection_pool.get()?;
+    let expected_current_slot = expected_current_slot();
+    // One slot is 12 sec, and we want the threshold to be 4 months before the current slot
+    let slot_threshold = expected_current_slot - (4 * 30 * 24 * 60 * 60 / 12);
```

Comment:
This time period should be more like 4.8 months. It is defined here https://github.com/ethereum/consensus-specs/blob/9d1e7087b3d0e5d3861b67dcf932158e26332fe3/specs/phase0/p2p-interface.md?plain=1#L206

```suggestion
    // One slot is 12 sec, and we want the threshold to be 4 months before the current slot
    let slot_threshold = expected_current_slot - (33042 * 32);
```

Reply:
There is no fixed time for the weak subjectivity period, it depends on network conditions (participation rate, slashing events, etc.) and market factors (e.g., how quickly they can buy, borrow, or bribe stakers). 

While the protocol doesn’t fix an official “weak subjectivity period,” the commonly referenced window for Ethereum mainnet today is around 2–4 months, so I think 4 months is a reasonable threshold.